lib/NotificationService.ts

```diff
@@ -0,0 +1,37 @@
+const https = require('https');
```

Comment:
Could you use an NPM plugin like https://www.npmjs.com/package/slack-webhook instead of building your own?